tests/testthat/test-printing.R

```diff
@@ -543,3 +543,54 @@ test_that("row label indentation is kept even if there are newline characters",
         "Found newline characters"
     )
 })
+
+test_that("Support for newline characters in all the parts", {
+    set.seed(1)
+    DM_trick <- DM %>% 
+        mutate(ARM2 = sample(c("TWO\nwords\n", "A wo\n\nrd"),
+                             replace = TRUE, nrow(DM))) # last \n is eaten up
+    levels(DM_trick$SEX)[3] <- "U\nN\nD\n"
+    tbl <- basic_table() %>% 
+        split_rows_by("SEX", split_label = "m\nannaggia\nsda\n",
+                      label_pos = "visible") %>% # last \n bug
+        split_cols_by("ARM2", split_label = "sda") %>% 
+        analyze("BMRKR1", na_str = "asd\nasd") %>%  # \n error
+        build_table(DM_trick)
+    
+    top_left(tbl) <- c("\na", "b\nd\n\n", "c\n\n") # last \n is eaten up, if in the middle error
```

Comment:
first \n are kept ->> note that if you want a blank line you could even do "\n    " as group of spaces are considered words now